service/templates/admin.html

```diff
@@ -490,7 +491,42 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 
             </section>
 
+
             <section layout="row" flex>
+
+              <md-card flex=50 id="ldap">
+                <md-card-title>
+                  <md-card-title-text>
+                    <span>LDAP Authentication</span>
+                  </md-card-title-text>
+                  <md-button ng-click="clearSection('auth','ldap')">
+                    <i class="fa fa-trash"></i>
+                  </md-button>
+                </md-card-title>
+                <md-card-content>
+                  <md-input-container class="control" style="width:45%;">
+                    <label>URL</label>
+                    <input type="text" ng-model="Settings.auth.crowd.url">
```

Comment:
These are bound to the crowd settings, not ldap

Reply:
fixed